examples/widgets/slider_demo.py

```diff
@@ -5,63 +5,83 @@
 
 Using the slider widget to control visual properties of your plot.
 
-In this example, a slider is used to choose the frequency of a sine
-wave. You can control many continuously-varying properties of your plot in
-this way.
+In this example, sliders are used to control the frequency and amplitude of
+a sine wave. You can control many continuously-varying properties of your plot
+in this way.
```

Comment:
```suggestion
In this example, sliders are used to control the frequency and amplitude of
a sine wave.
```

Now that there is a `valstep` argument this statement is limiting of the full potential of sliders.

Reply:
Also, it carries little information. We show two sliders, it is straight forward that this can be applied to different properties. --> Can be removed.